### ISSUE TITLE
Check and possibly remove multiple redundant invariant unit tests

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1493]
The invariant has two suites of unit tests.  One of them, sasinvariant/test/utest_data_handling.py, seems to have multiple redundancies (usually labelled incorrectly). In fact by redundancy it looks like exact copy paste of the code but labelled for example: low, lowq, very low q, low q with slit (there is NO slit smeared data!) etc). There is also a couple of tests I have commented out that should also be checked. 

TODO:
* I have described the problems in the doc strings with my suggestions about deleting.  However since it is about testing somebody else should check. 
* Also need to check whether removing the tests that the extrapolated Q* be less than 10% of Q* is reasonable.  For the invariant to be valid you would like that to be true but it is **not** in my opinion a feature of the invariant calculation at all and can quite  easily be larger depending on the data the user chooses to use.

**Comment by smk78** [comment]
There's a _unit test_ checking that the extrapolation is < 10% of Q*??? What's the point of that?

**Comment by butlerpd** [comment]
Indeed. I guess if you know that it should be true for the data you are using why not but then why not just say what it is like all the other tests.  I was going to see if somebody who is more persnickety than me about proper unit testing would comment on why that is useful :-) maybe @mdoucet ?
